jaide/core.py

```diff
@@ -212,7 +212,7 @@ def cli_to_shell(self):
 
     @check_instance
     def commit(self, commands="", confirmed=None, comment=None,
-               at_time=None, synchronize=False, req_format='text'):
+               at_time=None, synchronize=False, req_format='text', action="set", config_format="text"):
```

Comment:
should add these to the function comment, and list what the acceptable values are